model/app/registry.yaml

```diff
@@ -59,6 +59,14 @@ groups:
           The y (vertical) component of a screen coordinate, in screen pixels.
         stability: development
         examples: [ 12, 99 ]
+      - id: app.screen.name
+        type: string
+        stability: development
+        brief: The name of an application screen.
+        note: >
+          A screen represents a distinct view or page within an application, 
```

Comment:
I'm not sure "distinct view" is clear enough. Would that "view" have to cover the whole device's screen to be categorized as a screen? Or is it ok that there might be multiple "distinct views" on a device's screen at once, each one being considered a "screen" (which might happen on tablet UIs, for example)? I'm fine either way tbh, I just think we should make it clear here.